test/unit/test_mcp_server.py

```diff
@@ -0,0 +1,97 @@
+import json
+import os
+from typing import Any
+from unittest.mock import (
+    create_autospec,
+    patch,
+)
+
+import pytest
+from pyexasol import ExaConnection
+
+from exasol.ai.mcp.server.mcp_server import (
+    ENVAR_DSN,
+    ENVAR_PASSWORD,
+    ENVAR_SETTINGS,
+    ENVAR_USER,
+    get_mcp_settings,
+    main,
+)
+from exasol.ai.mcp.server.server_settings import McpServerSettings
+
+
+@pytest.fixture
+def clear_settings() -> None:
+    envar = os.environ.pop(ENVAR_SETTINGS, None)
```

Comment:
Probably mocking os.environ would be safer.

Reply:
I think it's safe enough, for a test anyway.